libs/core/src/lib/wizard/wizard-step/wizard-step.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="fd-wizard__step-wrapper">
+    <a
+        class="fd-wizard__step-container"
+        [attr.aria-label]="ariaLabel"
+        tabindex="0"
+        (click)="stepContainerKeypress()"
+        (keypress)="stepContainerKeypress($event)"
+    >
+        <ng-content select="fd-wizard-step-indicator"></ng-content>
+        <div class="fd-wizard__label-container" [ngClass]="optionalText ? 'fd-wizard__label-container--optional' : ''">
+            <div #wizardLabel>
+                <span class="fd-wizard__label">{{ label }}</span>
+                <span *ngIf="optionalText" class="fd-wizard__optional-text">{{ optionalText }}</span>
+            </div>
+        </div>
+    </a>
+    <span
+        *ngIf="!finalStep"
+        class="fd-wizard__connector"
+        [ngClass]="{
+            'fd-wizard__connector--active': status === 'completed',
+            'fd-wizard__connector--branching': branching
+        }"
+    >
+    </span>
```

Comment:
This component doesn't have a popover for stacked items, could you add popover for now and change it, when action sheet is finished?
![image](https://user-images.githubusercontent.com/26483208/95190325-6dff3a80-07cf-11eb-9f13-ff489220b3f9.png)

Reply:
We will address this in a separate issue